### DISCUSSION TITLE
Unable to deploy kafka cluster

### DISCUSSION THREAD

**Discussion by git-azeez** [strimzi/strimzi-kafka-operator#8366] [Q&A]
I have deployed a strimzi operator on my GKE cluster in a namespace. Operator POD is running fine.
But when I try to deploy kafka cluster, it says that cluster is deployed. However when I try to see the PODs there is not running kafka cluster pod in the current or any other namespace.

Please help me out to fix this issue

I have deployed strimzi operator using these helm charts and operator is deployed successfully
https://github.com/strimzi/strimzi-kafka-operator/tree/main/helm-charts/helm3/strimzi-kafka-operator

Now i used this file to deploy the kafka cluster using https://github.com/strimzi/strimzi-kafka-operator/blob/main/examples/kafka/kafka-jbod.yaml

**Comment by scholzj** [comment]
You will have to share the configurations, logs, the steps you did etc.

**Reply by git-azeez** [reply]
I have deployed strimzi operator using these helm charts and operator is deployed successfully
https://github.com/strimzi/strimzi-kafka-operator/tree/main/helm-charts/helm3/strimzi-kafka-operator


Now i used this file to deploy the kafka cluster using https://github.com/strimzi/strimzi-kafka-operator/blob/main/examples/kafka/kafka-jbod.yaml

**Reply by git-azeez** [reply]
I have deployed strimzi operator using these helm charts and operator is deployed successfully
https://github.com/strimzi/strimzi-kafka-operator/tree/main/helm-charts/helm3/strimzi-kafka-operator


Now i used this file to deploy the kafka cluster using https://github.com/strimzi/strimzi-kafka-operator/blob/main/examples/kafka/kafka-jbod.yaml

**Reply by scholzj** [reply]
Great ... but you still need to share the logs if nothing else.

**Reply by git-azeez** [reply]
Message: Failure executing: Get at https://192.168.208.1/apis/networking.k8s.io/v1/namespaces/mp-dev/networkpolicies/kafka-cluster-network-policy-zookeeper.
Message: Forbidden!COnfigured service account doesnt have access.Service account may have been revoked.networkpolicies.extensions "kafka-cluster-network-policy-zookeeper" is forbidden:
User "system:servuiceaccount:mp-dev:strimzi-cluster-operator" cannot get resource "networkpolicies" in API group extensions in namespaces mp-dev

**Reply by git-azeez** [reply]
That is the status of the kafka cluster. 
I have even created a network policy role and binded to the service account strimzi-cluster-operator. But still im getting that above error

current rolesbindings in this ns is
list-secrets-rolebinding
network-policy-reader-binding
strimzi-cluster-operator
strimzi-cluster-operator-entity-operator-delegation.


Earlier i was blocked stating that service account doesnot have secret get permission, then i created  this role list-secrets-rolebinding then this issue is fixed. But now im getting network issue

**Reply by scholzj** [reply]
That means that you installed it incorrectly and it is missing some RBAC resources. You should make sure it is installed correctly and has all the (cluster)roles and (cluster)rolebindings from the installation files. Adding some custom role bindings randomly based on the latest error is not the right way forward.

**Reply by git-azeez** [reply]
I have a limitation here which is i cant have or use cluster wide rbac, i have and can use roles only namespace level

**Reply by scholzj** [reply]
TBH, it is quite hard to help because you are not really sharing the information needed. You clearly don't have a regular installation. You did not share the actual log or the configuration. Without knowing all of these things, it is quite hard if not impossible to understand the issues and the possible solutions.
